lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -283,6 +283,17 @@
 
 #define PROCESS_A1toA1MemNonVar(name, func) PROCESS_A1toA1MemNonVar_COMMON(name, func,)
 
+#define PROCESS_SIZEtoA1MemNonVar_COMMON(name, func, suffix) \
```

Comment:
do you need to update jd.dll for this change?

Reply:
what is jd.dll?

---
In reply to: [205223580](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r205223580) [](ancestors = 205223580)